### DISCUSSION TITLE
Deprecated libraries

### DISCUSSION THREAD

**Discussion by Ramwells** [Gifford47/HCPBridgeMqtt#90] [General]
Hi -- firstly this is a big step forward from the previous HCP projects - I have been playing with the Horman door lifter for a couple of years and have adapted previous iterations to include MQTT and an Ethernet connection....

Anyway my query was are you going to update the libraries which now appear to be deprecated ?
I think these include:
AsyncElegantOTA, AsyncElegantOTA

Also I get several compile-time warnings which I guess you are aware of - including relating to ArduinoJSON :
class ARDUINOJSON_DEPRECATED("use JsonDocument instead"

Have you fixed this??

As I said - great work and thanks for sharing.

**Comment by Eagle3386** [comment]
@Ramwells Would you mind sharing how you got Ethernet working? I'm deliberately trying to keep my wireless devices count as low as possible, hence using Ethernet & PoE whenever possible. I'd love to see this bridge connected via 1 Ethernet cable only..
